components/postmark/actions/send-email-with-template/send-email-with-template.mjs

```diff
@@ -0,0 +1,51 @@
+import postmark from "../../postmark.app.mjs";
+
+let objSharedProps = {};
+postmark.methods.listSharedProps().forEach((propName) => {
+  objSharedProps[propName] = {
+    propDefinition: [
+      postmark,
+      propName,
+    ],
+  };
+});
+
+export default {
+  key: "postmark-send-email-with-template",
+  name: "Send an email with template",
+  description: "Send an email with Postmark using a template",
+  version: "0.1.23",
+  type: "action",
+  props: {
+    postmark,
+    template_alias: {
+      type: "string",
+      label: "Template",
+      description: "The template to use for this email.",
+      options() {
+        return this.postmark.listTemplates();
```

Comment:
Also is missing `async` keyword

Reply:
> We prefer stay the props that use `options()` in the app file.
I kept it here because only this action uses it, but if that's the standard I will move it to the app.

> Also is missing `async` keyword
I did not keep the async as `postmark.listTemplates` already returns a `Promise`, but will add it if that is a standard across options.